src/NamesList.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+class NamesList extends React.Component {
+  render() {
+    let namesLiList = [];
+    for (let i in this.props.namesList) {
```

Comment:
`for ... of` might be a little nicer here.

Reply:
Good suggestion, but unfortunately I'm going off the slides students saw. Code like this will eventually be replaced by `map` once we introduce that method. I'll make a note to introduce the `for...of` syntax though, thanks!